src/main/java/fr/openmc/core/features/contest/menu/ContributionMenu.java

```diff
@@ -93,7 +93,7 @@ public void onInventoryClick(InventoryClickEvent click) {
         );
 
         //TODO: itemadder dependency
-        Material shell_contest = CustomStack.getInstance("contest:contest_shell").getItemStack().getType();
+        Material shell_contest = CustomItemRegistry.getByName("omc_contest:contest_shell").getBest().getType();
```

Comment:
Ce nom de variable ne respecte pas les règles de contribution.
https://github.com/ServerOpenMC/PluginV2/blob/master/CONTRIBUTING.md#-style-de-code
```suggestion
        Material shellContest = CustomItemRegistry.getByName("omc_contest:contest_shell").getBest().getType();
```